packages/main/src/InputPopover.hbs

```diff
@@ -99,6 +99,7 @@
 				<ui5-li-groupheader data-ui5-key="{{key}}">{{{ this.text }}}</ui5-li-groupheader>
 			{{else}}
 				<ui5-li-suggestion-item
+					wrapping-Type="Normal"
```

Comment:
No need of capital letter for "wrapping-type"

Reply:
done